tests/behaviour/contracts/ElementaryTypeNames/type_name_type_removal.sol

```diff
@@ -0,0 +1,29 @@
+contract WARP {
+    struct s {
+        uint256 a;
+        uint256 b;
+    }
+
+    uint8 c;
+
+    function simple() public returns (int8) {
+        uint;
+        return 3; 
+    }
+
+    function indexAccess() public returns (int8) {
+        s[7][];
+        return 3;
```

Comment:
maybe the tests should return the unpacked values instead of just literals, to make sure they are getting unpacked correctly

Reply:
What we are trying to do here is remove the expression in line 15. We just want to see that it does not throw a transpilation error.